### DISCUSSION TITLE
Give us the ability to resell billing solutions licenses

### DISCUSSION THREAD

**Discussion by GreenArrowlol** [Paymenter/Paymenter#336] [Ideas]
Basically billing solutions like blesta and wisecp let us resell their licenses and I'd like to do that with paymenter. 

like https://blesta.com/resellers/

**Comment by SirHoDo** [comment]
Paymenter doesn't have purchasable licenses?

**Comment by GreenArrowlol** [comment]
I asked corwin and he said no
